lib/active_record/acts_as/relation.rb

```diff
@@ -76,10 +76,10 @@ def is_a?(klass)
           super || acting_as?(klass)
         end
 
-        def actable(options = {})
+        def actable(scope = nil, options = {})
           name = options.delete(:as) || :actable
```

Comment:
I checked CI and it looks like there's no Ruby <2.0 support required anymore. In that case, changing the method signature to `def actable(scope = nil, **options)` might do the trick, too?

Reply:
Maybe, but why? What's special about `scope` that it shouldn't be one of the `options`?

Reply:
The idea was to make it compatible with the signature of the usual `belongs_to` and `has_many` methods, I guess. So if one is already familiar with those, it would be more intuitive.

Reply:
Ah, I see! Yep, then we can change it (provided that tests are added).

Reply:
Thank you for reviewing. I 'll add the specs here later this week.